server/cloudinary/cloudinary.js

```diff
@@ -0,0 +1,12 @@
+const cloudinary = require('cloudinary').v2;
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+cloudinary.config({
+  cloud_name: process.env.CLOUDINARY_NAME,
+  api_key: process.env.CLOUDINARY_API_KEY,
+  api_secret: process.env.CLOUDINARY_API_SECRET,
```

Comment:
Remember to add these to the .env.template file!

Reply:
no not right now we need to discuss .. we dont have any api key so far.... it will only allow just one account so we need to have one common account for this

Reply:
The env.template file is just for us to know that we will need these keys in our private env files. The values can be added in later.

Reply:
ahhhhh got it